src/content/en/2020/markup.md

```diff
@@ -4,51 +4,58 @@ chapter_number: 3
 title: Markup
 description: Markup chapter of the 2020 Web Almanac covering general observations, the use of elements and attributes, as well as trivia and trends.
 authors: [j9t, catalinred, iandevlin]
-j9t_bio: Jens Oliver Meiert is a web developer and author (<a href="https://leanpub.com/css-optimization-basics"><cite>CSS Optimization Basics</cite></a>, <a href="https://leanpub.com/web-development-glossary"><cite>The Web Development Glossary</cite></a>), who works as an engineering manager at <a href="https://www.jimdo.com/">Jimdo</a>. He’s an expert on web development where he specializes in HTML and CSS optimization. Jens contributes to technical standards and regularly writes about his work and research, particularly on his website, <a href="https://meiert.com/en/">meiert.com</a>.
-# catalinred_bio: @@
-# iandevlin_bio: @@
+j9t_bio: Jens Oliver Meiert is a web developer and author (<a href="https://leanpub.com/css-optimization-basics"><cite>CSS Optimization Basics</cite></a>, <a href="https://leanpub.com/web-development-glossary"><cite>The Web Development Glossary</cite></a>), who works as an engineering manager at <a href="https://www.jimdo.com/">Jimdo</a>. He's an expert on web development where he specializes in HTML and CSS optimization. Jens contributes to technical standards and regularly writes about his work and research, particularly on his website, <a href="https://meiert.com/en/">meiert.com</a>.
+catalinred_bio: Catalin Rosu is a front-end developer at <a href="https://www.caphyon.com/">Caphyon</a> and currently works on <a href="https://www.wattspeed.com/">Wattspeed</a>. He has a passion for web standards and a keen eye for great UX & UI, things he <a href="https://twitter.com/catalinred">tweets</a> and writes about on <a href="https://catalin.red/">his website</a>.
+iandevlin_bio: Ian Devlin is a web developer who advocates for good, semantic HTML, as well as accessibility. He once wrote a book about <a href="https://www.peachpit.com/store/html5-multimedia-develop-and-design-9780321793935">HTML5 Multimedia</a>, and sporadically writes on <a href="https://iandevlin.com/">his website</a> about the Web and other things. He currently works as a Senior Frontend Engineer at <a href="https://www.real-digital.de/">real.digital</a> in Germany.
 reviewers: [zcorpan, matuzo, bkardell]
 analysts: [Tiggerito]
 translators: []
 discuss: 2039
 results: https://docs.google.com/spreadsheets/d/1Ta7amoUeaL4pILhWzH-SCzMX9PsZeb1x_mwrX2C4eY8/
 queries: 03_Markup
-featured_quote: We approach near-complete adoption of living HTML, are quick to prune our pages of fads, and we’re fast in adopting and shunning frameworks. And still, there are no signs we exhausted the options HTML gives us.
+featured_quote: We approach near-complete adoption of living HTML, are quick to prune our pages of fads, and we're fast in adopting and shunning frameworks. And still, there are no signs we exhausted the options HTML gives us.
```

Comment:
“Wrong” is a incorrect. “Not as elegant” definitely has some merit.

We are using the Google Developer Style Guide for the Web Almanac and [it recommends straight quote marks](https://developers.google.com/style/quotation-marks#straight-and-curly-quotation-marks):

> __Straight and curly quotation marks__
>
> Most typefaces support two forms of quotation marks and apostrophes: straight marks and curly, or typographic, marks. Some tools, like Google Docs, automatically convert straight quotation marks and apostrophes to the curly versions as you type. However, our guidance is to always use straight quotation marks and straight apostrophes, for these reasons:
> 
> - It's easy to get the direction of curly quotation marks (especially apostrophes) wrong. Using straight marks avoids this problem.
> - Code requires straight marks.
> - Many writers use coding tools (for example, Subversion) for working directly with markup. Because these tools do not produce curly marks by default, the writer must manually insert the marks, which is error prone.

So this is a conscious decision to follow this practice. I understand you both disagree with this convention and decision, but that is what we have gone with for the 2019 Web Almanac and that is what is listed in our [Editor’s Guide](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Editors'-Guide#suggested-process). We need to be consistent here.

I suggest you [raise a new issue for discussion](https://github.com/HTTPArchive/almanac.httparchive.org/issues) as to the pros and cons for this (and the effort to convert the 2019 Web Almanac!) but for now, and for this pull request, we should follow the convention already set and style guide we are using.

Reply:
Moving this discussion to https://github.com/HTTPArchive/almanac.httparchive.org/issues/1485